python_scripts/linear_models_sol_02.py

```diff
@@ -90,7 +90,7 @@
 # Now create a pipeline using `make_pipeline` consisting of a
 # `PolynomialFeatures` and a linear regression. Set `degree=2` and
 # `interaction_only=True` to the feature engineering step. Remember not to
-# include the bias to avoid redundancies with the linear's regression intercept.
+# include the bias to avoid redundancies with the linear regression intercept.
```

Comment:
```suggestion
# include a "bias" feature (that is a constant-valued feature) to avoid
# introducing a redundancy with the intercept of the subsequent linear
# regression model.
```